src/jit/bitsetasuint64.h

```diff
@@ -139,6 +144,11 @@ class BitSetOps</*BitSetType*/ UINT64,
         return (bs1 & bs2) == 0;
     }
 
+    static void LivenessD(Env env, UINT64& in, const UINT64 def, const UINT64 use, const UINT64 out)
```

Comment:
There is at least one use of each.
IsEmptyUnion: codegencommon.cpp:1079,
LivenessD: liveness.cpp:1193